web/annotation_editor_layer_builder.css

```diff
@@ -1217,6 +1217,7 @@
 
       @media (prefers-color-scheme: dark) {
         --example-color: #80808e;
+        --divider-color: #8f8f9d;
```

Comment:
Do we need this line here, since it's only for `dark` mode, given the "full" definitions further below?

Reply:
This seems to have accidentally re-appeared in the latest version of the patch.